docs/content/packages/webhook.md

```diff
@@ -1,15 +1,8 @@
 ---
 title: Incoming Webhooks
-permalink: /webhook
+slug: /webhook
```

Comment:
This changes the `webhook` documentation from the first to second:

1. https://tools.slack.dev/node-slack-sdk/packages/webhook
2. https://tools.slack.dev/node-slack-sdk/webhook

Which matches the path of other `packages` - the others don't include "packages" in the path!

Reply:
oh good catch -- you are indeed correct! permalink is what the old jekyll site used, i missed it

Reply:
Super interesting moves by `jekyll`, thanks for making this clear!! 🙏

Reply:
oh yes to wrap up explanation -- permalink is ignored entirely. when there's nothing set, docusaurus defaults to the folder structure the md file is in